[2025-02-02T18:09:29.051+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T18:09:29.064+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: northwind_etl.extract_orders manual__2025-02-02T18:09:27.557060+00:00 [queued]>
[2025-02-02T18:09:29.070+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: northwind_etl.extract_orders manual__2025-02-02T18:09:27.557060+00:00 [queued]>
[2025-02-02T18:09:29.071+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-02T18:09:29.082+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_orders> on 2025-02-02 18:09:27.557060+00:00
[2025-02-02T18:09:29.090+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'northwind_etl', 'extract_orders', 'manual__2025-02-02T18:09:27.557060+00:00', '--job-id', '411', '--raw', '--subdir', 'DAGS_FOLDER/northwind_etl/northwind_etl.py', '--cfg-path', '/tmp/tmpdigfcuhi']
[2025-02-02T18:09:29.092+0000] {standard_task_runner.py:105} INFO - Job 411: Subtask extract_orders
[2025-02-02T18:09:29.093+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=201) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-02T18:09:29.094+0000] {standard_task_runner.py:72} INFO - Started process 203 to run task
[2025-02-02T18:09:29.127+0000] {task_command.py:467} INFO - Running <TaskInstance: northwind_etl.extract_orders manual__2025-02-02T18:09:27.557060+00:00 [running]> on host a0f93d34c5b4
[2025-02-02T18:09:29.192+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gabriel' AIRFLOW_CTX_DAG_ID='northwind_etl' AIRFLOW_CTX_TASK_ID='extract_orders' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T18:09:27.557060+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T18:09:27.557060+00:00'
[2025-02-02T18:09:29.193+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T18:09:29.216+0000] {logging_mixin.py:190} INFO - table: orders, date: 2025-02-02
[2025-02-02T18:09:29.340+0000] {logging_mixin.py:190} INFO - Extraction saved on: /opt/***/data/postgres/orders/2025-02-02/orders.parquet
[2025-02-02T18:09:29.340+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T18:09:29.347+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T18:09:29.348+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=northwind_etl, task_id=extract_orders, run_id=manual__2025-02-02T18:09:27.557060+00:00, execution_date=20250202T180927, start_date=20250202T180929, end_date=20250202T180929
[2025-02-02T18:09:29.389+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T18:09:29.400+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T18:09:29.402+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
